workflows/diagnostics/fv3net/diagnostics/offline/views/create_report.py

```diff
@@ -417,14 +433,8 @@ def create_report(args):
     logger.info(f"Save report to {args.output_path}")
 
     # Gcloud logging allows metrics to get ingested into database
-    gcloud_logging = {
-        "json": metrics,
-        "logging.googleapis.com/labels": {
-            "model": metadata["model_path"],
-            "commit_sha": metadata.get("commit", "n/a"),
-        },
-    }
-    print(json.dumps(gcloud_logging))
+    print(json.dumps({"json": metrics}))
+    _print_step_metadata_json(args, metadata)
```

Comment:
do we not want to attach the step metadata to the streamed json as labels?

i.e. just print one dict that is something like

```
{
    'json': metrics,
    'logging.googleapis.com/labels": {k: v for k, v in step_metadata.asdict()}
}
```

Reply:
I believe all the "facts" will be tied to the step level metadata through `name: pod.metadata.step_name` or `pod.name` as the primary key.  IIRC, labels have some restrictions to make them easily searchable, which might be too constraining for some of this metdata.

@AnnaKwa, maybe the step metadata dump should have a unique key so the payload would look like {"step_metdata": <Metadata JSON>} instead of having it all at the top level.  This would make it easy to query with something like jsonPayload.step_metadata as a field to select on.